### ISSUE TITLE
palette: how to create an extended pallete

### ISSUE THREAD

**Issue by bolt12** [cdepillabout/termonad#84]
I'm having a hard time figuring how to create a custom ExtendedPallete.

I have the this:

```Haskell
fst8 = [(sRGB24 19 19 19), (sRGB24 122  99 64), (sRGB24 145 122 90), (sRGB24 70 38 20), (sRGB24 108 49 8), (sRGB24 113 103 51), (sRGB24 71 65 32), (sRGB24 197 114 0)]
snd8 = [(sRGB24 19 19 19), (sRGB24 122 99 64), (sRGB24 145 122 90), (sRGB24 70 38 20) , (sRGB24 108 49 8), (sRGB24 113 103 51), (sRGB24 126 116 63), (sRGB24 197 114 0)]
```

But i can't convert it to a `Vec N8 c`, I dont have access to the `Vec` constructor...
Maybe create an end-user list2Vec function?

**Comment by bolt12** [comment]
I get this error: > Module ‘Termonad.Config.Vec’ does not export ‘unsafeFromListVec_’

I can export it myself, but for the sake of this example maybe you forgot to edit that on the `Vec.hs`

**Comment by cdepillabout** [comment]
@bolt12 Hmm, are you sure you did a `git pull`?

The current version of `master` has the above code in `Termonad.Config.Vec`:

https://github.com/cdepillabout/termonad/blob/master/src/Termonad/Config/Vec.hs#L394-L395

There is no export list in `Termonad.Config.Vec`, so everything should be exported.

**Comment by bolt12** [comment]
Yes, I'm sure!

Look at my screenshot
![image](https://user-images.githubusercontent.com/20261421/48659824-2a6e7c80-ea4f-11e8-9aaf-18ae18b5364e.png)

I got it working by just building the Vec with `(:*)` and pass it to the ExtendedPalette.

**Comment by cdepillabout** [comment]
Hmm, this is really strange.

Have you done a `stack build` after doing `git pull` but before the `stack exec -- termonad`?  Assuming that is so, I have no idea why this would be happening.

Running `stack exec -- termonad` should use the termonad binary that was created the last time you ran `stack build`.  It appears that some other `termonad` binary is being used instead.  I'm not sure why this would be the case.

I'm glad you were able to get it working with the `(:*)` constructor though.

**Comment by bolt12** [comment]
You're right! I forgot to `stack build`!

All good now, thanks for your help!

**Comment by cdepillabout** [comment]
I released a new version of termonad (1.0.1.0) that has these new functions in it:

http://hackage.haskell.org/package/termonad-1.0.1.0/docs/Termonad-Config-Vec.html#v:fromListVec_

Thanks for bringing this up, @bolt12.
